bindings/python/examples/how_tos/advanced_transactions/claim_transaction.py

```diff
@@ -29,5 +29,5 @@
 transaction = account.claim_outputs(output_ids)
 print(f'Transaction sent: {transaction.transactionId}')
 
-block_id = account.retry_transaction_until_included(transaction)
+block_id = account.reissue_transaction_until_included(transaction)
```

Comment:
this should also be fixed to transaction.transactionId right?

Reply:
Yes, done